enterprise_gateway/services/processproxies/processproxy.py

```diff
@@ -369,10 +371,21 @@ def _create_ssh_tunnel(self, port_name, local_port, remote_port, remote_ip, ssh_
         self.log.debug("Creating SSH tunnel for '{}': 127.0.0.1:'{}' to '{}':'{}'"
                        .format(port_name, local_port, remote_ip, remote_port))
         try:
-            tunnel.ssh_tunnel(local_port, remote_port, ssh_server, remote_ip, ssh_key, timeout=maxint)
+            process = self._spawn_ssh_tunnel(port_name, local_port, remote_port, remote_ip, ssh_server, ssh_key=None)
+            self.tunnel_processes.append(process)
         except Exception as e:
             raise RuntimeError("Could not open SSH tunnel for port {}. Exception: '{}'".format(port_name, e))
 
+    def _spawn_ssh_tunnel(self, port_name, local_port, remote_port, remote_ip, ssh_server, ssh_key=None):
+        """ This method spawns a child process to create a SSH tunnel and returns the spawned process.
+        """
+        if sys.platform == 'win32':
+            return tunnel.paramiko_tunnel(local_port, remote_port, ssh_server, remote_ip, ssh_key, timeout=maxint)
+        else:
+            cmd = "%s -S none -L 127.0.0.1:%i:%s:%i %s sleep %i" % (
```

Comment:
Since these implementations clearly use `127.0.0.1`, I think we should remove the [single use of `tunnel_ip`](
https://github.com/jupyter-incubator/enterprise_gateway/pull/201/files#diff-71b92028d190d942e7926650b50d055cL405) and replace it with `127.0.0.1` (and remove its declaration).

Reply:
Done